static_src/components/navbar.jsx

```diff
@@ -82,17 +82,19 @@ export class Nav extends React.Component {
 
   render() {
     const mainList = this.styler('usa-sidenav-list', 'sidenav-list', 'sidenav-level-one');
-    const secondList = this.styler('usa-sidenav-list', 'sidenav-list', 'sidenav-level-two');
-    const thirdList = this.styler('sidenav-list', 'usa-sidenav-sub_list', 'sidenav-level-three');
+    const secondList = this.styler('usa-sidenav-sub_list', 'sidenav-list', 'sidenav-level-two');
+    const thirdList = this.styler('sidenav-list', 'sidenav-level-three');
     const downArrow = this.styler('menu-arrow', 'sidenav-arrow', 'sidenav-arrow-down');
     const rightArrow = this.styler('menu-arrow', 'sidenav-arrow', 'sidenav-arrow-right');
     const header = this.styler('sidenav-header');
+    const faHome = this.styler('fa', 'fa-home');
```

Comment:
I really don't want to bring font awesome into the dashboard. We're only using it on docs right now and would prefer to remove if from there rather then add it to more places. Is that what we're attempting to do here with the `fa` class?